### PR TITLE
inserting file into another folder

### PR DESCRIPTION
**Pull request by KseniaLL**
file is inserted in a new folder now

### CODE REVIEW

File: project/GoogleDriveV3.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    previous_parents = ",".join(file.get('parents'))
+    try:
+        # Move the file to the new folder
+        file = service.files().update(fileId=file_id,
+                                            addParents=folder_id,
+                                            removeParents=previous_parents,
+                                            fields='id, parents').execute()
+    except googleapiclient.errors.HttpError:
+        print("Could notmove a file to the excpected folder. Check the folder ID in your config.yaml file")
```
Comment (haggaret) [inline project/GoogleDriveV3.py]: missing space between not and move
typo in expected

**Review by haggaret — APPROVED** [review]

File: project/GoogleDriveV3.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    service = build('drive', 'v3', http=creds.authorize(Http()))
+
+    return service
+
+def moveFileToFolder(fileID, folderID):
+
+    global file
+
+    service = buildService(file)
```
Comment (dnorth98) [inline project/GoogleDriveV3.py]: Don't call the buildService function in each function.  Have the caller call it before calling each "functional" method.

Comment (dnorth98) [inline project/GoogleDriveV3.py]: We will want to error check that the service is valid in the action function and error out accordingly

File: project/spreadsheet.py [hunk 3]
```diff
@@ -257,6 +256,17 @@ def initialize(email, configMap, arg):
 
             SPREADSHEET_ID = sheetCreated['spreadsheetId']
 
+            if 'spreadsheet_database' in configMap:
+                folderID = configMap['spreadsheet_database']['folderID']
+            else:
+                print("Spreadsheet_database section is missing in your config file")
+                return None
+
```
Comment (dnorth98) [inline project/spreadsheet.py:264]: Setup the service first and then pass the service object into the moveFileToFolder function

**Review by dnorth98 — CHANGES_REQUESTED** [review]

**Review by dnorth98 — APPROVED** [review]

**Pull request merged**